ContactFactoryNew.cls

```diff
@@ -0,0 +1,48 @@
+/**
+* @description: This is the data factory for contact object where dummy records 
+                are created and inserted and furthur used in the test class.
+* @author: Noopur Naik
+*
+* @revision:
+* Version              Description
+* -----------------------------------
+* 1.0               Initial revision.
+**/
+@isTest
+Public class ContactFactoryNew {
```

Comment:
Why class name is like this? You are suppose to follow proper naming convention.